docs/platforms/python/integrations/bottle/index.mdx

```diff
@@ -90,6 +91,22 @@ def myendpoint():
 - If you set `transaction_style="endpoint"`, the transaction name is `"myendpoint"`, since that is the route handler function's name.
 - If you set `transaction_style="url"`, the transaction name is `"/myurl/<foo>"`, since that is the URL path.
 
+
+### `failed_request_status_codes`
+
+A `set` of integers that will determine when an [`HTTPResponse`](https://bottlepy.org/docs/dev/api.html#bottle.HTTPResponse), which is raised or returned by the request handler, should be reported to Sentry. The `HTTPResponse` is reported to Sentry if its status code is contained in the `failed_request_status_codes` set.
+
+Examples of valid values for `failed_request_status_codes`:
+
+- `{500}` will only report `HTTPResponse` with status 500.
+- `{400, *range(500, 600)}` will report `HTTPResponse` with status 400 as well as those in the 5xx range.
+- `set()` (the empty set) will not report any `HTTPResponse` to Sentry.
+
+The default is `{*range(500, 600)}`, meaning that any `HTTPResponse` with a status in the 5xx range is reported to Sentry.
+
+Regardless of how `failed_request_status_codes` is set, any exceptions raised by the handler, which are not of type `HTTPResponse` (or a subclass) are reported to Sentry. For example, if your request handler raises an unhandled `AttributeError`, the `AttributeError` gets reported to Sentry, even if you have set `failed_request_status_codes=set()`.
```

Comment:
```suggestion
Regardless of how `failed_request_status_codes` is configured, any exceptions raised by the handler which are not of type `HTTPResponse` (or a subclass) are reported to Sentry. For example, if your request handler raises an unhandled `AttributeError`, the `AttributeError` gets reported to Sentry, even if you have set `failed_request_status_codes=set()`.
```

Reply:
I agree that "configured" is better here; however, if I remember my English classes correctly 😅, the comma needs to be there. However, I did notice that I am missing a comment after the "(or a subclass)," so I will add this.

Reply:
Actually, I think I will rewrite this and just say that any "non-`HTTPResponse`" exceptions are reported